### PR TITLE
[hi] Add content/hi/docs/tasks/_index.md

### PR DESCRIPTION
**Pull request by rajatgupta24**
Hindi localization of the file: content/hi/docs/tasks/_index.md

PR is linked under this issue #29353

/language hi

### CODE REVIEW

**Comment by Atharva-Shinde** [comment]
LGTM
@daminisatya @mittalyashu

File: content/hi/docs/tasks/_index.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: कार्य
+main_menu: true
+weight: 50
+content_type: concept
+---
+
+<!-- overview -->
+
+कुबेरनेट्स प्रलेखन के इस खंड में ऐसे पृष्ठ हैं जो दिखाते हैं कि व्यक्तिगत कार्य कैसे करें। एक कार्य पृष्ठ दिखाता है कि किसी एक काम को कैसे करना है, आमतौर पर चरणों का एक छोटा क्रम देकर।
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/_index.md:10]:
```suggestion
कुबेरनेट्स प्रलेखन के इस खंड के पृष्ठ एकल कार्य करने का तरीका दिखाते हैं। आमतौर पर, कार्य पृष्ठ दिखाता है कि किसी एक काम को कई छोटे चरणों में विभाजित करके कैसे करना है।
```

File: content/hi/docs/tasks/_index.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+यदि आप एक कार्य पृष्ठ लिखना चाहते हैं, तो देखें
+[दस्तावेज़ीकरण के लिए एक पीआर बनाएं ](/docs/contribute/new-content/open-a-pr/).
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/_index.md:13]:
```suggestion
यदि आप एक कार्य पृष्ठ लिखना चाहते हैं, तो 
[प्रलेखन के लिए एक पुल अनुरोध बनाएं](/docs/contribute/new-content/open-a-pr/) देखें।
```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]

**Comment by anubha-v-ardhan** [comment]
Thank you @rajatgupta24

**Pull request merged**